tests/scripts/list-identifiers.sh

```diff
@@ -8,6 +8,7 @@ if [ -d include/mbedtls ]; then :; else
 fi
 
 HEADERS=$( ls include/mbedtls/*.h | egrep -v 'compat-1\.3\.h|bn_mul' )
+HEADERS="$HEADERS 3rdparty/everest/include/everest/everest.h 3rdparty/everest/include/everest/x25519.h"
```

Comment:
Those two files are new, hand-written additions (the interface to the Everest code), i.e. those contain `mbedtls_*` function names that those scripts need/want to track. I added those bits to the scripts directly as there's no real infrastructure for tests/checks in `3rdparty`, but in the long run it would be a good idea to move those to the respective directories, I think.

Reply:
Ok. We've been planning to rewrite `check-names` and friends for a while to make them less brittle (but it's pretty low on our priority list), we'll tackle those extra files when we get around to it.